lighthouse-core/audits/accessibility/frame-title.js

```diff
@@ -11,6 +11,19 @@
  */
 
 const AxeAudit = require('./axe-audit');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if all frame elements on the page have a title HTML attribute to describe their contents. This descriptive title is shown to users when no user action is required. */
```

Comment:
`Title of an accesibility audit that evaluates if all '<frame>' and '<iframe>' elements on the page have a title HTML attribute to describe their contents.`

Reply:
done